xgcm/grid.py

```diff
@@ -1387,6 +1416,8 @@ def cumint(self, da, axis, **kwargs):
         axis : str, list of str
```

Comment:
Could you replace this here with `%(grid_func.parameters)s` aswell. Sorry for the trouble.